src/pygaps/characterisation/enth_sorp_whittaker.py

```diff
@@ -2,21 +2,27 @@
 
 import numpy as np
 import scipy.constants
+import warnings
 
 import pygaps.modelling as pgm
-from pygaps import logger
 from pygaps.core.baseisotherm import BaseIsotherm
 from pygaps.core.modelisotherm import ModelIsotherm
 from pygaps.core.pointisotherm import PointIsotherm
 from pygaps.utilities.exceptions import CalculationError
 from pygaps.utilities.exceptions import ParameterError
+from pygaps.core.adsorbate import Adsorbate
+from pygaps.graphing.calc_graphs import isosteric_enthalpy_plot
+from pygaps import logger
 
 
 def enthalpy_sorption_whittaker(
-    isotherm: "BaseIsotherm",
+    isotherm: BaseIsotherm,
+    branch: str = 'ads',
     model: str = 'Toth',
     loading: list = None,
     verbose: bool = False,
+    dographs: bool = True,
```

Comment:
would not have a separate graphing parameter

Reply:
[657de27](https://github.com/pauliacomi/pyGAPS/pull/54/commits/657de27e74a01b1bf050dc0304c64dafa673d178)